.github/workflows/testing.yml

```diff
@@ -11,6 +11,7 @@ jobs:
         uses: actions/checkout@v2.3.4
       - name: Install dependencies
         run: |
+          sudo apt-get update
```

Comment:
Is this necessary as part of this PR?  I think you might be trying to address two things at the same time. I'd keep the documentation stuff separate from the build issues.

Reply:
good point, split that out to a separate PR